mlu_op.h

```diff
@@ -3074,6 +3074,65 @@ mluOpLog(mluOpHandle_t handle,
          const mluOpTensorDescriptor_t y_desc,
          void *y);
 
+// Group: Log
+/*!
+ * @brief Returns a one-dimensional tensor of \b steps points logarithmically 
+ * spaced with base \b base between \b base^start and \b base^end.
+ * 
+ * @param[in] handle
+ * Handle to a Cambricon MLU-OPS context that is used to manage MLU devices and
+ * queues in the log operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] start
+ * The starting value for the set of points.
+ * @param[in] end
+ * The ending value for the set of points.
+ * @param[in] steps
+ * Number of points to sample between \b start and \b end.
+ * @param[in] base
+ * Base of the logarithm function.
+ * @param[in] res_desc
+ * The descriptor of the tensor \b res. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[out] res
+ * Pointer to the MLU memory that stores the output tensor \b res.
+ *
+ * @par Return
+ * - ::MLUOP_STATUS_SUCCESS, ::MLUOP_STATUS_BAD_PARAM
+ *
+ * @par Data Type
+ * - The data types of input tensor and output tensor should be the same.
+ * - The supported data types of input and output tensors are as follows:
+ *   - output tensor: half, float, bfloat16, int32
+ *
+ * @par Data Layout
+ * - None.
+ *
+ * @par Scale Limitation
+ * - \b start, \b end, \b base could not be NAN or INFINITY.
+ * - \b base greater than 0, \b steps greater than 0.
+ * - \b steps less than or equal to the length of the output array \b res.
```

Comment:
```suggestion
 * - \b steps should be less than or equal to the length of the output array \b res.
```
上面用的是output tensor，请统一。

Reply:
已修改